Make_Disk_Image.mk

```diff
@@ -0,0 +1,24 @@
+FAT_SECTORS:=65536
+DISK_SECTORS:=69632
+DISK_START:=2048
+DISK_END:=67584
+
+TEMP_IMG := temp.img
+PARTED := sudo parted
+PARTED_PARAMS := -s -a minimal
+
+partition.img: $(FILES)
+	dd if=/dev/zero of=$(TEMP_IMG) bs=512 count=$(FAT_SECTORS)
+	mformat -i $(TEMP_IMG) -h 32 -t 32 -n 64 -c 1 ::
+	mcopy -i $(TEMP_IMG) $(FILES) ::
+	cp $(TEMP_IMG) $@
+	rm $(TEMP_IMG)
+
+hd.img: partition.img
+	dd if=/dev/zero of=$(TEMP_IMG) bs=512 count=$(DISK_SECTORS)
+	$(PARTED) $(TEMP_IMG) $(PARTED_PARAMS) mklabel gpt
+	$(PARTED) $(TEMP_IMG) $(PARTED_PARAMS) mkpart EFI FAT16 $(DISK_START)s $(DISK_END)s
```

Comment:
`mkpart part-type [fs-type] start end`
Make a part-type partition with filesystem fs-type (if specified), beginning at start and ending at end (by default in megabytes). 
fs-type can be one of "fat16", "fat32", "ext2", "HFS", "linux-swap", "NTFS", "reiserfs", or "ufs". 
part-type should be one of "primary", "logical", or "extended".